config/webpack.common.js

```diff
@@ -353,28 +353,32 @@ module.exports = function (options) {
          * File loader for supporting fonts, for example, in CSS files.
          */
         {
-          test: /\.woff2?$|\.ttf$|\.eot$|\.svg$/,
-          use: [
-            {
-              loader: 'url-loader',
-              query: {
-                limit: 3000,
-                name: '_assets/fonts/[name].' + (isProd ? '[hash]' : '') + '[ext]'
-              }
+          test: /\.(woff2||woff|ttf|eot|svg)$/,
```

Comment:
What does the double pipe do here?

Reply:
that is an error - fixed with the latest update